other/mutate-large-termination-gps/kyverno-test.yaml

```diff
@@ -0,0 +1,19 @@
+name: mutate-termination-grace-period-seconds
+policies:
+  - mutate-large-termination-gps.yaml
+resources:
+  - resource.yaml
+results:
+  - policy: mutate-termination-grace-period-seconds
+    rule: mutate-termination-grace-period-seconds
+    resource: demo-pod01
+    patchedResource: patchedResource1.yaml
+    kind: Pod
+    result: pass
+  - policy: mutate-termination-grace-period-seconds
+    rule: mutate-termination-grace-period-seconds
+    resource: demo-pod02
+    patchedResource: patchedResource2.yaml
+    kind: Pod
+    result: skip  
```

Comment:
This shouldn't be scored as a `skip` yet the CI tests are succeeding and I don't know why. It appears based on the policy they should fail.

Reply:
Not sure about this @chipzoller , I thought as default tGPS in k8s is 30 sec so it will get skipped as I haven't mentioned it in my resources.

Reply:
Nevermind. Kubernetes API server will add `30` by default which is why it doesn't meet preconditions.

Reply:
You are correct, Oshi. I was wrong here.